src/init.js

```diff
@@ -0,0 +1,57 @@
+import inquirer from 'inquirer';
+import fs from 'fs';
+import logger from './logger';
+
+const questions = [
+  {
+    type: 'input',
+    name: 'gridUrl',
+    message:
+      'What is your selenium grid Url? \n  Example: http://selenium-grid:4444/wd/hub\n'
+  },
+  {
+    type: 'input',
+    name: 'baselineDir',
+    message: 'Where is your baseline image directory? \n  Example: ./baseline\n'
```

Comment:
Maybe, 'Where would you like your baseline image directory? \n Default is ./baseline'

Reply:
Same with the other directory questions.